code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -1,9 +1,21 @@
 // "Useful" items - I'm guessing things that might be used at work?
 /datum/gear/utility
-	display_name = "briefcase"
-	path = /obj/item/storage/briefcase
+	display_name = "camera"
+	path = /obj/item/camera
```

Comment:
camera?

Reply:
changing base /datum/gear/utility to be camera so that the briefcase selection can be a child of it, otherwise every utility entry would inherit the selection

Reply:
A filthy antipattern, but permissible since I don't want to unbreak it right now.